File: courier_application/app/src/main/java/by/bstu/vs/stpms/courier_application/model/network/dto/UserDto.java
package by.bstu.vs.stpms.courier_application.model.network.dto;

import lombok.Data;

import java.util.Collection;
import java.util.Set;

@Data
public class UserDto extends AbstractDto {
    private String firstName;
    private String secondName;
    private String email;
    private String phone;
    private String password;
    private String confirmPassword;
    private Collection<Long> ordersId;
    private Set<RoleDto> roles;
}

File: courier_application/app/src/main/java/by/bstu/vs/stpms/courier_application/model/entity/UserDto.kt
package by.bstu.vs.stpms.courier_application.model.entity

class UserDto (
    private val firstName: String,
    private val secondName: String,
    private val email: String,
    private val phone: String,
    private val password: String,
    private val confirmPassword: String?,
    private val ordersId: Collection<Long>,
    private val roles: Set<RoleDto>
) : AbstractDto()